internal/scylla/client.go

```
@@ -136,10 +136,43 @@ func (c *Client) doHttpRequest(req *http.Request) (resp *http.Response, temporar
 		return
 	}
 
-	temporaryErr = resp.StatusCode == http.StatusBadGateway || resp.StatusCode == http.StatusGatewayTimeout || resp.StatusCode == http.StatusTooManyRequests
+	if getErrorCode(resp) == 000001 {
+		return resp, true, fmt.Errorf("too many request response")
+	}
+	temporaryErr = temporaryErr || resp.StatusCode == http.StatusBadGateway || resp.StatusCode == http.StatusGatewayTimeout || resp.StatusCode == http.StatusTooManyRequests
 	return
 }
 
+func getBodyData(resp *http.Response) []byte {
+	if resp.Body == nil {
+		return nil
+	}
+	data, _ := io.ReadAll(resp.Body)
```

please don't ignore errors

---

done